docroot/themes/custom/uids_base/scss/webforms/webforms.scss

```diff
@@ -1,12 +1,226 @@
 @import "uids/assets/scss/_variables.scss";
 @import "uids/assets/scss/_utilities.scss";
 
-.webform-terms-of-service-details {
-  max-height: 80vh !important;
+
+.webform-submission-form {
+  label {
+    font-size: $base-font-size;
+    font-weight: $font-weight-bold;
+    line-height: 1.6;
+  }
+  .fieldgroup {
+    border: none;
+    padding: 0;
+    & .fieldset-legend {
+      display: block;
+      font-size: $base-font-size;
+      font-weight: $font-weight-bold;
+      line-height: 1.6;
+    }
+    .form-item {
+      margin-top: 0.125rem;
+      margin-bottom: 0.125rem;
+      margin-block: 0.125rem;
+    }
+  }
+
+  // --------------------------------
+  // Help/Description 
+  // --------------------------------
+  .help-text,
+  .description,
+  .field-prefix,
+  .field-suffix {
+    font-size: 0.9rem;
+    line-height: 1.4;
+  }
+  .description {
+    margin: $xsm 0;
+  }
+  input + .description,
+  input + .field-prefix,
+  input + .field-suffix {
+    position: relative;
+    top: -0.75rem;
+    inset-block-start: -0.75rem;
+  }
+
+  // --------------------------------
+  // More
+  // --------------------------------
+  .webform-element-more {
+    &--content {
+      line-height: 1.6;
+    }
+  }
+  .description .webform-element-more {
+    margin-top: $xsm;
+    margin-block-start: $xsm;
+  }
+
+  // --------------------------------
+  // Checkboxes
+  // --------------------------------
+  [type='checkbox']+label,
+  [type='radio']+label {
+    font-weight: $font-weight-normal;
+  }
+  .webform-type-checkboxes,
+  .webform-checkboxes-other,
+  .webform-type-radios,
+  .webform-type-webform-radios-other {
+    & .fieldset-wrapper {
+      margin-left: $sm;
+      margin-inline-start: $sm;
+    }
+  }
+
+  // --------------------------------
+  // Details
+  // --------------------------------
+  .details-description {
+    line-height: 1.6;
+  }
+  
+  // --------------------------------
+  // Flexbox
+  // --------------------------------
+  .webform-flexbox {
+    & .webform-flex--container {
+      & .form-actions [type="submit"] {
+        width: 100%; // Set button full-width.
+        height: 57px;
+      }
+    }
+    & .form-item {
```

Comment:
@bewoldt can you take a look at these flexbox layouts on https://sandbox.uiowa.ddev.site/webform-layouts and try removing these lines and look at the spacing issue?